images/imaging.py

```diff
@@ -243,7 +245,7 @@ def create_plot(
     """
 
     fig, ax = plt.subplots(figsize=(8, 8), dpi=image_attributes.resolution // 8)
-    fig.subplots_adjust(0, 0, 1, 1)
```

Comment:
yes, this was for the colorbar, other options are of course fine as well

Reply:
I will merge this as is for now as you are the one mainly using these images, but in the future we should move to having them on top of the plot (as otherwise the grid that the image is created on is not the same as the output pixel grid)